docs/src/components/MarkdownPage/AdditionalLinks.js

```diff
@@ -99,6 +99,20 @@ const AdditionalLinks = (props: Props) => {
           </BpkLink>
         )}
 
+        {/* Web npm link */}
+        {platform && platform === PLATFORMS.web && githubPath && (
```

Comment:
To reduce duplication, we use the `githubPath` metadata item for the NPM link, because we know they match on web components.